lightning/src/ln/channelmanager.rs

```diff
@@ -6903,7 +6903,8 @@ mod tests {
 
 		// Next, attempt a keysend payment and make sure it fails.
 		let params = RouteParameters {
-			payee: Payee::for_keysend(expected_route.last().unwrap().node.get_our_node_id()),
+			payee: Payee::for_keysend(expected_route.last().unwrap().node.get_our_node_id())
+				.with_max_total_cltv_expiry_delta(TEST_MAX_TOTAL_CLTV_EXPIRY_DELTA),
```

Comment:
Do we need to modify all these tests now when it's not explicitly being tested?

Reply:
No, probably not. :)